src/view_dna_n_to_random.hpp

```diff
@@ -49,11 +51,11 @@ struct dna_n_to_random_fn
         {
             if (seqan3::to_char(in) == 'N')
             {
-                return seqan3::assign_rank_to(rng() % 4, seqan3::dna4{});
+                return seqan3::assign_rank_to(rng() % TAlph::alphabet_size, TAlph{});
```

Comment:
```suggestion
                return seqan3::assign_rank_to(rng() % seqan3::alphabet_size<TAlph>, TAlph{});
```

Please use global traits in generic contexts.